httomo/method_wrappers/dezinging.py

```diff
@@ -30,18 +30,30 @@ def __init__(
         **kwargs,
     ):
         super().__init__(
-            method_repository, module_path, method_name, comm, save_result, output_mapping, **kwargs
+            method_repository,
+            module_path,
+            method_name,
+            comm,
+            save_result,
+            output_mapping,
+            **kwargs,
         )
         assert (
-            method_name == "remove_outlier3d"
-        ), "Only remove_outlier3d is supported at the moment"
+            method_name == "remove_outlier"
+        ), "Only remove_outlier is supported at the moment"
         self._flats_darks_processed = False
 
     def execute(self, block: DataSetBlock) -> DataSetBlock:
         self._gpu_time_info = GpuTimeInfo()
         # check if data needs to be transfered host <-> device
         block = self._transfer_data(block)
 
+        args = self._build_kwargs(self._transform_params(self._config_params), block)
+        try:
+            self._config_params["axis"] = args["axis"]
+        except:
+            pass
+
```

Comment:
Sorry, just to be clearer, I was suggesting something like the following:
```python
        args = self._build_kwargs(self._transform_params(self._config_params), block)
 
         with catch_gputime() as t:
            block.data = self.method(block.data, **args)
            if not self._flats_darks_processed:
                block.darks = self.method(block.darks, **args)
                block.flats = self.method(block.flats, **args)
                self._flats_darks_processed = True
```

where the code inside `with` block below has been updated to pass `**args` to all executions of `self.method()` in place of `**self._config_params`, so then projections/daks/flats are processed with the same updated parameters in `args`?